app/build.gradle

```diff
@@ -1,11 +1,14 @@
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'kotlin-kapt'
 
 android {
     compileSdkVersion 29
 
     def versionMajor = 4
     def versionMinor = 3
-    def versionPatch = 0
+    def versionPatch = 1
```

Comment:
Sorry, we're on 4.3.2 now :/

I needed to fix the group making settings schema migration.